sh.exe.stackdump

```diff
@@ -0,0 +1,3 @@
+Stack trace:
```

Comment:
I'm not sure what this file is...

Reply:
Yeah, well I keep getting random files generated such as `package-lock.json`

Reply:
@indianbhaji `package-lock.json` is coming from `npm`.  that is a newer feature they added to compete with yarn.  that is expected and should be included in source control.

This file, however, I've not ever seen before.  Do you know where its coming from?

Reply:
maybe the shell you're using? Especially if it crashes sometimes.